homework/style.css

```diff
@@ -1,3 +1,127 @@
-.alert-error {
```

Comment:
On a mobile phone the boxes appear to be clipped on the right-hand side.

![image](https://user-images.githubusercontent.com/2788771/61803769-11666880-ae33-11e9-8e06-cd8b426905de.png)

Reply:
Fixed the styling problem on mobile screens. It now shows as expected